doc/requirements_and_expectations.md

```diff
@@ -0,0 +1,30 @@
+# Requirements for SQL Query Builder
+
+## 1. Functional Requirements
+
+- Provide an interface for selecting tables from the database.
+- Enable creating and configuring relationships (`JOIN`) between selected tables.
```

Comment:
Again, why you still use word "relationship" you are not configuring "relationships" you can only select joins